xml/schema/Core/Address.xml

```diff
@@ -13,6 +13,11 @@
     <uniqueName>address_id</uniqueName>
     <type>int unsigned</type>
     <export>true</export>
+    <usage>
+      <import>false</import>
+      <export>true</export>
+      <duplicate_matching>false</duplicate_matching>
```

Comment:
I don't want to have to set tests going again now - but will add `token` to this when I do a metadata pass